source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -49,6 +49,7 @@
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
```

Comment:
Why do we need this? Do we need to `pip install django-cors-headers` for it to work? Do we need to add `corsheaders` in `INSTALLED_APP`?

Reply:
We already have all other step of this setup, we just forgot to add the Middleware :

**requirements.txt**
`django-cors-headers==2.0.2`

**settings.py**
```
INSTALLED_APPS = [
...
    'corsheaders',
...
]
```
[Package's documentation](https://github.com/ottoyiu/django-cors-headers/)
[Why use cors](https://fr.wikipedia.org/wiki/Cross-origin_resource_sharing)

Reply:
Oh right, sorry I should have looked at the code myself.